### DISCUSSION TITLE
0x50 error when updating Zooz 7.15 --> 7.17

### DISCUSSION THREAD

**Discussion by muddro1** [kpine/zwave-js-server-docker#17] [Q&A]
I don't know if you would be able to assist, but trying to update my Zooz controller to the latest firmware. 

I initially went the simplicity studio route to update, where I got 0x50 errors. Communications with zooz thus far have not been helpful. I came along your wiki and tried to update utilizing your method, and after I select the firmware in minicom, the transfer never completes:
![image](https://user-images.githubusercontent.com/15880479/155212383-893a3fc7-2a0f-41bb-9152-a00e990ee967.png)

I saw note #2 on https://community.silabs.com/s/article/z-wave-700-otw-of-controller?language=en_US, but not sure how to do that, and if that is even the problem. Any help is appreciated.

**Comment by kpine** [comment]
From your screenshot it doesn't appear as if you are getting to the bootloader? Do you run the printf command to enter the bootloader, and do you ever see the bootloader prompt?

```
Gecko Bootloader v1.5.1
1. upload gbl
2. run
3. ebl info
BL >
begin upload
CCCC
```

**Comment by muddro1** [comment]
I ran both printf commands. There was no output from those commands. I do see the bootloader prompt:
![image](https://user-images.githubusercontent.com/15880479/155219266-a3775dfe-4fad-474f-933c-da2c4eeb1770.png)

**Comment by kpine** [comment]
Just to confirm, you are using file `ZW_SerialAPI_Controller_7_17_1_344_ZGM130S_REGION_US.gbl`? The important part is the ZGM130S. 

If that's the case, unfortunately I won't be of much help. The incomplete transfer error might be the equivalent to the PC Controller 0x50 error. I know both of those errors happened to me when I originally tried to use the wrong firmware. For the 7.17.1 update I just used PC Controller with my Zooz.

Zooz has [official instructions](https://www.support.getzooz.com/kb/article/931-how-to-perform-an-ota-firmware-update-on-your-zst10-700-z-wave-stick/) to upgrade using PC Controller. I would try following that step-by-step (if you haven't already), just to make Zooz customer support happy, and if it doesn't work go back to them.

**Comment by kpine** [comment]
[According to](https://github.com/zwave-js/node-zwave-js/issues/2545#issuecomment-1049550432) another commenter, Zooz has identified an issue with some sticks, and says you can either mail back yours for them to fix, or wait for a new firmware version that also supposedly fixes it. I'd continue to engage with support in that case.

**Reply by muddro1** [reply]
I think you are correct. Have been in contact with Zooz and they want me to send it back. I suppose I can break out my old controller and upload the current nvm to it while they fix the zooz?
